JSDemos/Demos/PivotGrid/SimpleArray/description.md

```diff
@@ -1 +1,8 @@
-The PivotGrid is&nbsp;a&nbsp;high-performance, enterprise-ready UI component optimized for multi-dimensional data analysis. It&nbsp;allows your end users to&nbsp;easily arrange and summarize information in&nbsp;a&nbsp;cross-tabular format. In&nbsp;this demo, the PivotGrid&rsquo;s simplicity of&nbsp;use is&nbsp;illustrated. Sorting and filtering are enabled so&nbsp;you can sort data in&nbsp;ascending or&nbsp;descending order and by&nbsp;summary values. You can drag and drop fields to&nbsp;reorganize the PivotGrid and change filtering properties to&nbsp;analyze data more efficiently.
\ No newline at end of file
+The PivotGrid component can display data from an array of objects. Use the [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource) property to bind the PivotGrid to data. This property accepts a [PivotGridDataSource](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/) instance or its configuration object. Assign your array to this object's [store](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/store/) property.
+
+To display data in the PivotGrid, assign an array to the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) property. Each object in this array configures a single pivot grid field. Set the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to populate the pivot grid field with data.
+
+Fields can belong to four different areas: row, column, filter, and data. To specify the area, set the [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) property. Fields that do not belong to any area are displayed in the [field chooser](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/).
```

Comment:
> Fields can belong to four different areas: row, column, filter, and data.

Do these areas serve as parts of the PivotGrid component?

> To specify the area, set the [area](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#area) property.

Can I distribute fields between the target areas or do I have to bind fields from the same source to one specific area? This is not quite clear.

Reply:
A summary from our personal discussion:

Write this:
> You can distribute fields between four different areas: row, column, filter, and data.

Instead of
> Fields can belong to four different areas: row, column, filter, and data.